packages/client/examples/node.js/storeCar.js

```diff
@@ -9,15 +10,17 @@ async function main() {
   const storage = new NFTStorage({ endpoint, token })
 
   // locally chunk'n'hash the file to get the CID and pack the blocks in to a CAR
-  const { root, car } = await packToBlob({
+  const { root, out } = await pack({
```

Comment:
The generic `pack` is more efficient because we do not need to consume the entire content to produce the Blob. The Blob is useful for the browser, but I would really prefer to point people to use the `CarReader`, as (specially if they look into `packToFs`) will require way less memory.

What do you think?

Reply:
Ah gotcha - the [`CarReader` class](https://github.com/ipld/js-car#carreader) reads the entire thing into memory. You need to use the indexed reader instead.

Reply:
oh, I did not know that about the CarReader. I will change it to the indexed one